Removes the dagger line form the jarjar file which is known th cause issues when depending on rules kotlin from source.

diff --git a/src/main/kotlin/shade.jarjar b/src/main/kotlin/shade.jarjar
index d320d3d..85ae26e 100644
--- a/src/main/kotlin/shade.jarjar
+++ b/src/main/kotlin/shade.jarjar
@@ -1,3 +1,2 @@
-rule dagger.** io.bazel.kotlin.builder.dagger.@1
 rule com.google.common.** io.bazel.kotlin.builder.guava.@1
 zap kotlin.**
